src/client/directory.ts

```diff
@@ -10,60 +10,150 @@
  */
 
 /**
- * Type definitions that is are wherever the API expects the ID or of an application to be passed. We have both, since
- * internally appId is used to identify app, but externally we use name to repsect specification
+ * Type alias to indicate when an Application Identifier should be passed. Application Identifiers
+ * are described [here](https://fdc3.finos.org/docs/1.0/appd-discovery#application-identifier).
  *
- * It is not possible for TypeScript to verify that only valid application names/appIds are passed
- * to the FDC3 API. This type definition is more a hint to the callee, over a plain "string" argument.
+ * This type alias exists to disambiguate raw string app identity from the [[AppName]].
  */
 export type AppId = string;
+/**
+ * Type alias to indicate when an Application Name should be passed. This will be a human-readable
+ * name that will be displayed on screen.
+ *
+ * This type alias exists to disambiguate raw string app identity from the [[AppId]].
+ */
 export type AppName = string;
 
 /**
- * An application in the app directory
+ * An application in the app directory.
  */
 export interface Application {
+    /**
```

Comment:
These properties come out ordered alphabetically in the docs. Not sure if there is some way in typedoc to preserve the ordering of interfaces or if that's even something we want, but thought I'd raise it for discussion.

Reply:
Yeah fair. Looks like you [can't change ordering anyway.](https://github.com/TypeStrong/typedoc/issues/112)

Reply:
Ok, as long as we're happy with the look.